production/helm/loki/templates/bloom-gateway/statefulset-bloom-gateway.yaml

```
@@ -53,6 +53,10 @@ spec:
       hostAliases:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- with .Values.bloomGateway.dnsConfig }}
```

What did you think about this? Have a common value which can be overridden by more global one?

```suggestion
      {{- with default .Values.bloomGateway.dnsConfig | default .Values.loki.dnsConfig }}
```

---

yes, excellent idea. will do.